packages/allure-mocha/src/MochaAllureReporter.ts

```diff
@@ -81,6 +81,9 @@ export class MochaAllureReporter extends Mocha.reporters.Base {
   }
 
   private onHookEnd(hook: Mocha.Hook): void {
-    this.coreReporter.endHook(hook.error());
+    if(typeof(hook.error) === 'function')
+      this.coreReporter.endHook(hook.error());
```

Comment:
Could we set default value for `hook.error`? Like just stub function to avoid additional conditions?
Or we can use optional chaining operator to reach the same result:
```js
this.coreReporter.endHook(hook?.error?.());
```

Reply:
@epszaw I will give it a try later in the month when I get some more time. Thanks for the feedback.

Reply:
Hi @epszaw I just got back around to looking at this.  It looks like we can close this pull request.  I can see you added this change in this commit:  https://github.com/allure-framework/allure-js/commit/cbb34269c9ddcc6510e065693dfeb9b8c853beaf.  Thanks for adding it in.